### DISCUSSION TITLE
Search results vs autocomplete results.

### DISCUSSION THREAD

**Discussion by Bashev** [Smile-SA/elasticsuite#2929] [General]
I have few customers which start complains about search results and i have no idea how to proceed because of that.

Expectations of the customers are to see the same results (or most like the same) which are proposed in auto complete list in the "Results Page".

Most of the time in auto complete they see results with partial matching, but this partial matching returns nothing or totally different results in "Results Page", because of Relevance settings.

Do you have any suggestions how to make both results similar.

**Comment by romainruaud** [comment]
See #970 and #2811 for some explanations. But to keep it short, that's two different queries.
